tests/Database/Table/Selection.order().phpt

```diff
@@ -14,12 +14,29 @@ Nette\Database\Helpers::loadFromFile($connection, __DIR__ . "/../files/{$driverN
 
 test(function() use ($context) {
 	$apps = array();
-	foreach ($context->table('book')->where('title LIKE ?', '%t%')->order('title')->limit(3) as $book) {  // SELECT * FROM `book` WHERE (`title` LIKE ?) ORDER BY `title` LIMIT 3
+
+	$selection = $context->table('book')->where('title LIKE ?', '%t%')->order('title DESC')->limit(3);
```

Comment:
I've added also `DESC` keyword to previous test because without it result was the same as defined order in SQL fixture.